index.html

```diff
@@ -1796,7 +1796,9 @@ <h3>Presentations</h3>
 The contents of the <code>verifiableCredential</code> <a>property</a> shown
 above are <a>verifiable credentials</a>, as described by this specification. The
 contents of the <code>proof</code> <a>property</a> are proofs, as described by
-the Linked Data Proofs [[?LD-PROOFS]] specification.
+the Linked Data Proofs [[?LD-PROOFS]] specification. An example of a
+<a>verifiable presentation</a> using the JWT proof mechanism is given in section
+<a href="#json web token"></a>.
```

Comment:
This is an invalid fragment identifier... change to `#json-web-token`.

Reply:
Done